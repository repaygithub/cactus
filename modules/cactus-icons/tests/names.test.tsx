import fs from 'fs'
import path from 'path'

describe('Icon names', (): void => {
  test('should match the snapshot of icon names', (): void => {
    const names = fs.readdirSync(path.join(__dirname, '../i'))
    expect(names).toMatchSnapshot()
  })
})
