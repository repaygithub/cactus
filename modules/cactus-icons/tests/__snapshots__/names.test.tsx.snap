// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon names should match the snapshot of icon names 1`] = `
Array [
  "actions-add.d.ts",
  "actions-add.js",
  "actions-add.js.map",
  "actions-coachmarks.d.ts",
  "actions-coachmarks.js",
  "actions-coachmarks.js.map",
  "actions-copy.d.ts",
  "actions-copy.js",
  "actions-copy.js.map",
  "actions-delete.d.ts",
  "actions-delete.js",
  "actions-delete.js.map",
  "actions-download.d.ts",
  "actions-download.js",
  "actions-download.js.map",
  "actions-edit.d.ts",
  "actions-edit.js",
  "actions-edit.js.map",
  "actions-filter.d.ts",
  "actions-filter.js",
  "actions-filter.js.map",
  "actions-gear.d.ts",
  "actions-gear.js",
  "actions-gear.js.map",
  "actions-key.d.ts",
  "actions-key.js",
  "actions-key.js.map",
  "actions-print.d.ts",
  "actions-print.js",
  "actions-print.js.map",
  "actions-redo.d.ts",
  "actions-redo.js",
  "actions-redo.js.map",
  "actions-refresh.d.ts",
  "actions-refresh.js",
  "actions-refresh.js.map",
  "actions-remove.d.ts",
  "actions-remove.js",
  "actions-remove.js.map",
  "actions-undo.d.ts",
  "actions-undo.js",
  "actions-undo.js.map",
  "actions-upload.d.ts",
  "actions-upload.js",
  "actions-upload.js.map",
  "batchstatus-inprogress.d.ts",
  "batchstatus-inprogress.js",
  "batchstatus-inprogress.js.map",
  "batchstatus-open.d.ts",
  "batchstatus-open.js",
  "batchstatus-open.js.map",
  "batchstatus-reconciled.d.ts",
  "batchstatus-reconciled.js",
  "batchstatus-reconciled.js.map",
  "batchstatus-settle.d.ts",
  "batchstatus-settle.js",
  "batchstatus-settle.js.map",
  "descriptive-assistant.d.ts",
  "descriptive-assistant.js",
  "descriptive-assistant.js.map",
  "descriptive-at.d.ts",
  "descriptive-at.js",
  "descriptive-at.js.map",
  "descriptive-bars.d.ts",
  "descriptive-bars.js",
  "descriptive-bars.js.map",
  "descriptive-box.d.ts",
  "descriptive-box.js",
  "descriptive-box.js.map",
  "descriptive-calendar.d.ts",
  "descriptive-calendar.js",
  "descriptive-calendar.js.map",
  "descriptive-cart.d.ts",
  "descriptive-cart.js",
  "descriptive-cart.js.map",
  "descriptive-clock.d.ts",
  "descriptive-clock.js",
  "descriptive-clock.js.map",
  "descriptive-cloud.d.ts",
  "descriptive-cloud.js",
  "descriptive-cloud.js.map",
  "descriptive-comment.d.ts",
  "descriptive-comment.js",
  "descriptive-comment.js.map",
  "descriptive-connect.d.ts",
  "descriptive-connect.js",
  "descriptive-connect.js.map",
  "descriptive-discussions.d.ts",
  "descriptive-discussions.js",
  "descriptive-discussions.js.map",
  "descriptive-document1.d.ts",
  "descriptive-document1.js",
  "descriptive-document1.js.map",
  "descriptive-document2.d.ts",
  "descriptive-document2.js",
  "descriptive-document2.js.map",
  "descriptive-envelope.d.ts",
  "descriptive-envelope.js",
  "descriptive-envelope.js.map",
  "descriptive-filter.d.ts",
  "descriptive-filter.js",
  "descriptive-filter.js.map",
  "descriptive-folder.d.ts",
  "descriptive-folder.js",
  "descriptive-folder.js.map",
  "descriptive-home.d.ts",
  "descriptive-home.js",
  "descriptive-home.js.map",
  "descriptive-internet.d.ts",
  "descriptive-internet.js",
  "descriptive-internet.js.map",
  "descriptive-location.d.ts",
  "descriptive-location.js",
  "descriptive-location.js.map",
  "descriptive-lock.d.ts",
  "descriptive-lock.js",
  "descriptive-lock.js.map",
  "descriptive-piechart.d.ts",
  "descriptive-piechart.js",
  "descriptive-piechart.js.map",
  "descriptive-presentation.d.ts",
  "descriptive-presentation.js",
  "descriptive-presentation.js.map",
  "descriptive-profile.d.ts",
  "descriptive-profile.js",
  "descriptive-profile.js.map",
  "descriptive-receipt.d.ts",
  "descriptive-receipt.js",
  "descriptive-receipt.js.map",
  "descriptive-ruler.d.ts",
  "descriptive-ruler.js",
  "descriptive-ruler.js.map",
  "descriptive-suitcase.d.ts",
  "descriptive-suitcase.js",
  "descriptive-suitcase.js.map",
  "descriptive-tray.d.ts",
  "descriptive-tray.js",
  "descriptive-tray.js.map",
  "iconSizes.d.ts",
  "iconSizes.js",
  "iconSizes.js.map",
  "index.cjs.js",
  "index.cjs.js.map",
  "index.d.ts",
  "index.js",
  "index.js.map",
  "interface-card.d.ts",
  "interface-card.js",
  "interface-card.js.map",
  "interface-desktop.d.ts",
  "interface-desktop.js",
  "interface-desktop.js.map",
  "interface-laptop.d.ts",
  "interface-laptop.js",
  "interface-laptop.js.map",
  "interface-page.d.ts",
  "interface-page.js",
  "interface-page.js.map",
  "interface-sms.d.ts",
  "interface-sms.js",
  "interface-sms.js.map",
  "interface-tables.d.ts",
  "interface-tables.js",
  "interface-tables.js.map",
  "multimedia-call.d.ts",
  "multimedia-call.js",
  "multimedia-call.js.map",
  "multimedia-camera.d.ts",
  "multimedia-camera.js",
  "multimedia-camera.js.map",
  "multimedia-videocamera.d.ts",
  "multimedia-videocamera.js",
  "multimedia-videocamera.js.map",
  "multimedia-voice.d.ts",
  "multimedia-voice.js",
  "multimedia-voice.js.map",
  "multimedia-volume.d.ts",
  "multimedia-volume.js",
  "multimedia-volume.js.map",
  "navigation-arrow-down.d.ts",
  "navigation-arrow-down.js",
  "navigation-arrow-down.js.map",
  "navigation-arrow-left.d.ts",
  "navigation-arrow-left.js",
  "navigation-arrow-left.js.map",
  "navigation-arrow-right.d.ts",
  "navigation-arrow-right.js",
  "navigation-arrow-right.js.map",
  "navigation-arrow-up.d.ts",
  "navigation-arrow-up.js",
  "navigation-arrow-up.js.map",
  "navigation-chevron-down.d.ts",
  "navigation-chevron-down.js",
  "navigation-chevron-down.js.map",
  "navigation-chevron-left.d.ts",
  "navigation-chevron-left.js",
  "navigation-chevron-left.js.map",
  "navigation-chevron-nav.d.ts",
  "navigation-chevron-nav.js",
  "navigation-chevron-nav.js.map",
  "navigation-chevron-right.d.ts",
  "navigation-chevron-right.js",
  "navigation-chevron-right.js.map",
  "navigation-chevron-up.d.ts",
  "navigation-chevron-up.js",
  "navigation-chevron-up.js.map",
  "navigation-circle-down.d.ts",
  "navigation-circle-down.js",
  "navigation-circle-down.js.map",
  "navigation-circle-left.d.ts",
  "navigation-circle-left.js",
  "navigation-circle-left.js.map",
  "navigation-circle-right.d.ts",
  "navigation-circle-right.js",
  "navigation-circle-right.js.map",
  "navigation-circle-up.d.ts",
  "navigation-circle-up.js",
  "navigation-circle-up.js.map",
  "navigation-close.d.ts",
  "navigation-close.js",
  "navigation-close.js.map",
  "navigation-drag&drop.d.ts",
  "navigation-drag&drop.js",
  "navigation-drag&drop.js.map",
  "navigation-hamburger.d.ts",
  "navigation-hamburger.js",
  "navigation-hamburger.js.map",
  "navigation-less.d.ts",
  "navigation-less.js",
  "navigation-less.js.map",
  "navigation-logout.d.ts",
  "navigation-logout.js",
  "navigation-logout.js.map",
  "navigation-menu-dots.d.ts",
  "navigation-menu-dots.js",
  "navigation-menu-dots.js.map",
  "navigation-plus.d.ts",
  "navigation-plus.js",
  "navigation-plus.js.map",
  "navigation-search.d.ts",
  "navigation-search.js",
  "navigation-search.js.map",
  "notification-alert.d.ts",
  "notification-alert.js",
  "notification-alert.js.map",
  "notification-error.d.ts",
  "notification-error.js",
  "notification-error.js.map",
  "notification-info.d.ts",
  "notification-info.js",
  "notification-info.js.map",
  "notification-note.d.ts",
  "notification-note.js",
  "notification-note.js.map",
  "payment-amex.d.ts",
  "payment-amex.js",
  "payment-amex.js.map",
  "payment-bank.d.ts",
  "payment-bank.js",
  "payment-bank.js.map",
  "payment-barcode.d.ts",
  "payment-barcode.js",
  "payment-barcode.js.map",
  "payment-batch.d.ts",
  "payment-batch.js",
  "payment-batch.js.map",
  "payment-card-back.d.ts",
  "payment-card-back.js",
  "payment-card-back.js.map",
  "payment-cash.d.ts",
  "payment-cash.js",
  "payment-cash.js.map",
  "payment-cashback.d.ts",
  "payment-cashback.js",
  "payment-cashback.js.map",
  "payment-charge.d.ts",
  "payment-charge.js",
  "payment-charge.js.map",
  "payment-credit.d.ts",
  "payment-credit.js",
  "payment-credit.js.map",
  "payment-csc.d.ts",
  "payment-csc.js",
  "payment-csc.js.map",
  "payment-cvv.d.ts",
  "payment-cvv.js",
  "payment-cvv.js.map",
  "payment-debit.d.ts",
  "payment-debit.js",
  "payment-debit.js.map",
  "payment-invoice.d.ts",
  "payment-invoice.js",
  "payment-invoice.js.map",
  "payment-master.d.ts",
  "payment-master.js",
  "payment-master.js.map",
  "payment-pay.d.ts",
  "payment-pay.js",
  "payment-pay.js.map",
  "payment-paybycard.d.ts",
  "payment-paybycard.js",
  "payment-paybycard.js.map",
  "payment-payment.d.ts",
  "payment-payment.js",
  "payment-payment.js.map",
  "payment-refund.d.ts",
  "payment-refund.js",
  "payment-refund.js.map",
  "payment-token.d.ts",
  "payment-token.js",
  "payment-token.js.map",
  "payment-wallet.d.ts",
  "payment-wallet.js",
  "payment-wallet.js.map",
  "status-approve.d.ts",
  "status-approve.js",
  "status-approve.js.map",
  "status-check.d.ts",
  "status-check.js",
  "status-check.js.map",
  "status-danger.d.ts",
  "status-danger.js",
  "status-danger.js.map",
  "status-follow.d.ts",
  "status-follow.js",
  "status-follow.js.map",
  "status-like.d.ts",
  "status-like.js",
  "status-like.js.map",
  "status-money.d.ts",
  "status-money.js",
  "status-money.js.map",
  "status-recomend.d.ts",
  "status-recomend.js",
  "status-recomend.js.map",
  "status-spinner.d.ts",
  "status-spinner.js",
  "status-spinner.js.map",
  "status-star.d.ts",
  "status-star.js",
  "status-star.js.map",
  "status-sun.d.ts",
  "status-sun.js",
  "status-sun.js.map",
  "status-target.d.ts",
  "status-target.js",
  "status-target.js.map",
  "status-thumbsdown.d.ts",
  "status-thumbsdown.js",
  "status-thumbsdown.js.map",
  "status-thumbsup.d.ts",
  "status-thumbsup.js",
  "status-thumbsup.js.map",
  "status-trophy.d.ts",
  "status-trophy.js",
  "status-trophy.js.map",
  "status-unfollow.d.ts",
  "status-unfollow.js",
  "status-unfollow.js.map",
  "types.d.ts",
  "types.js",
  "types.js.map",
]
`;
